analyzer-comments/elixir/log-level/use_cond.md

```diff
@@ -0,0 +1,6 @@
+# use `cond`
+
+Use [`cond`] in both `to_label` and `alert_recipient`. [`cond`] allows to check
```

Comment:
I would like to give a justification for this suggestion. Something similar to what we wrote for other concept exercises:
```suggestion
The purpose of this exercise is to teach [`cond`]. Use [`cond`] in both `to_label` and `alert_recipient`. [`cond`] allows to check
```